test/com/google/javascript/jscomp/parsing/NewParserTest.java

```diff
@@ -538,10 +532,9 @@ public void testJSDocAttachment5() {
    * propagate to following code due to {@link JSDocInfo} aggregation.
    */
   public void testJSDocAttachment6() throws Exception {
```

Comment:
OK, that's good. I just made a general comment for the tests being rewritten here. If some of them can be deleted, just go ahead and remove them.